frontend/src/views/c-authorship.vue

```diff
@@ -99,108 +99,19 @@
-          span.ignored-tag.fileTypeLabel(
-            v-if='file.isIgnored'
-          ) ignored ({{ file.lineCount }}) &nbsp;
-          span.icons
-            a(
-              v-bind:class="!isBrokenLink(getHistoryLink(file)) ? '' : 'broken-link'",
-              v-bind:href="getHistoryLink(file)", target="_blank"
-            )
-              .tooltip(
-                v-on:mouseover="onTitleTooltipHover(`${file.path}-view-history-tooltip`, `${file.path}-title`)",
-                v-on:mouseout="resetTitleTooltip(`${file.path}-view-history-tooltip`, `${file.path}-title`)"
-              )
-                font-awesome-icon.button(icon="history")
-                span.tooltip-text(
-                  v-bind:ref="`${file.path}-view-history-tooltip`"
-                ) {{getLinkMessage(getHistoryLink(file), 'Click to view the history view of file')}}
-            a(
-              v-if='!file.isBinary',
-              v-bind:class="!isBrokenLink(getBlameLink(file)) ? '' : 'broken-link'",
-              v-bind:href="getBlameLink(file)", target="_blank",
-              title="click to view the blame view of file"
-            )
-              .tooltip(
-                v-on:mouseover="onTitleTooltipHover(`${file.path}-view-blame-tooltip`, `${file.path}-title`)",
-                v-on:mouseout="resetTitleTooltip(`${file.path}-view-blame-tooltip`, `${file.path}-title`)"
-              )
-                font-awesome-icon.button(icon="user-edit")
-                span.tooltip-text(
-                  v-bind:ref="`${file.path}-view-blame-tooltip`"
-                ) {{getLinkMessage(getBlameLink(file), 'Click to view the blame view of file')}}
-          .author-breakdown(v-if="info.isMergeGroup")
-            .author-breakdown__legend(
-              v-for="author in getAuthors(file)",
-              v-bind:key="author"
-            )
-              font-awesome-icon(
-                icon="circle",
-                v-bind:style="{ 'color': authorColors[author] }"
-              )
-              span &nbsp; {{ author }} &nbsp;
-        pre.file-content(v-if="file.isBinary", v-show="file.active")
-          .binary-segment
-            .indicator BIN
-            .bin-text Binary file not shown.
-        pre.file-content(v-else-if="file.isIgnored", v-show="file.active")
-          .ignored-segment
-            .ignore-text File is ignored.
-        pre.hljs.file-content(v-else-if="file.wasCodeLoaded", v-show="file.active")
-          c-segment-collection(v-bind:segments="file.segments", v-bind:path="file.path")
+    template(v-for="(file, index) in selectedFiles", v-bind:key="file.path")
+      c-authorship-file(v-bind:file="file", v-bind:index="index",
+        v-bind:files-sort-type="this.filesSortType", v-bind:info="this.info",
+        v-bind:author-colors="this.authorColors", v-bind:file-type-colors="this.fileTypeColors",
+        @toggle-file-active-property="(file) => toggleFileActiveProperty(file)")
```

Comment:
Is there a reason you wrote it like this instead of 
```
 @toggle-file-active-property="toggleFileActiveProperty"
```
Just curious, as I thought they were equivalent

Reply:
I wasn't sure if this syntax worked so I did it as a Vue tutorial suggested. I agree your way of writing it looks cleaner, so I'll use that instead.